apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -2733,11 +2737,13 @@ const deleteChosenConversationsEpic: AppEpic = (action$, state$) =>
             folders: folders.filter(
               (folder) =>
                 !fullyChosenFolderIds.includes(`${folder.id}/`) &&
-                conversations.some((c) => c.id.startsWith(`${folder.id}/`)),
+                (conversations.some((c) => c.id.startsWith(`${folder.id}/`)) ||
+                  emptyFoldersIds.some((id) => id === folder.id)),
             ),
           }),
         ),
         of(ConversationsActions.resetChosenConversations()),
+        of(ConversationsActions.resetChosenEmptyFolders()),
```

Comment:
Why do we need separate action for that? We can do it into `resetChosenConversations`

Reply:
I think it's better to implement separate actions for improved separation of concerns, testability, and reusability.

Reply:
These actions aren't used separately and it's bad practice to dispatch a lot of actions into the same reducer instead do all stuff on one action. This action means "reset all selections". Why do we need to dospatch two of them in each places?

Reply:
Code changed as suggested.
Thank you for the explanation!